architecture/mce-overview-ocp.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="mce-overview-ocp"]
+= About multicluster engine for Kubernetes operator
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: mce-overview-ocp
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator. The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available as part of Red Hat Advanced Cluster Management for Kubernetes, or as a standalone operator that is supported as part of your OpenShift or Red Hat OpenShift Kubernetes Engine subscription.
+
+If you use Red Hat Advanced Cluster Management, multicluster engine is included and provides the following capabilities:
+
+* klusterlet agent, which registers managed clusters to the hub.
+* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of OpenShift, such as SNO on bare metal. The Infrastructure Operator includes Central Infrastructure Management (CIM), which provides an interactive user interface in the Red Hat Advanced Cluster Management console where you can provision clusters.
+* Open cluster management, which provides resources to manage multiple Kubernetes clusters.
+
+If you use {product-title} 4.8.2 or later, you can separately install multicluster engine to access the following capabilities:
+
+* Hive, which provisions and manages self-managed OpenShift clusters to the hub.
+* Zero touch provisioning (ZTP), which provides full automation for cluster creation on bare metal and vSphere with minimal intervention. ZTP also offers a GitOps workflow for managing deployments.
+* Support for hosted control planes for {product-title}.
+
+The operator is entitled with a Red Hat OpenShift support subscription and is delivered separately from the core OpenShift payload. To start to use mulicluster engine, you must deploy the OpenShift cluster and then install the operator. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multicluster_engine_overview#mce-install-intro[the installation documentation].
+
+[id="mce-additional-resources-ocp"]
+== Additional resources
+
+For the complete documentation for multicluster engine, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/index[multicluster engine documentation], which is part of the product documentation for Red Hat Advanced Cluster Management. 
```

Comment:
Should we also include a note to the effect of:
"If you are entitled to Red Hat Advanced Cluster Management you can deploy the operator and get full management capabilities.  See the Red Hat Advanced Cluster Management install doc topic <link>"